### PR TITLE
LiisaPortfolioTracker_2

### PR DESCRIPTION
**Pull request by paulogustavopeixoto**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [ ] This pull request is being made by the same account as the accepted application.
- [x] I have disclosed any and all sources of reused code in the submitted repositories and have done my due diligence to meet its license requirements.
- [x] In case of acceptance, invoices must be submitted and payments will be transferred to the Polkadot AssetHub and/or fiat account provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://grants.web3.foundation/docs/Support%20Docs/milestone-deliverables-guidelines).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/1855

### CODE REVIEW

**Review by keeganquigley — CHANGES_REQUESTED** [review]
Hi @paulogustavopeixoto I'm confused as to why you are trying to submit this as an evaluation. Is this intended to be a delivery? Typically deliveries are submitted by the original [PR author](https://github.com/LiisaNFT) to the [deliveries](https://github.com/w3f/Grant-Milestone-Delivery/tree/master/deliveries) folder.

Additionally, the application linked above appears to be the wrong one, I believe the correct grant is https://github.com/w3f/Grants-Program/pull/1855 can you please fix this?

**Comment by paulogustavopeixoto** [comment]
Hey @keeganquigley thanks for the the feedback. I will update the delivery to the correct folder. I have to submit with this account since the original PR author we converted the account into an organization on github.

**Comment by paulogustavopeixoto** [comment]
> Hi @paulogustavopeixoto I'm confused as to why you are trying to submit this as an evaluation. Is this intended to be a delivery? Typically deliveries are submitted by the original [PR author](https://github.com/LiisaNFT) to the [deliveries](https://github.com/w3f/Grant-Milestone-Delivery/tree/master/deliveries) folder.
> 
> Additionally, the application linked above appears to be the wrong one, I believe the correct grant is [w3f/Grants-Program#1855](https://github.com/w3f/Grants-Program/pull/1855) can you please fix this?

Hey @keeganquigley . Made the necessary changes. Removed the delivery file from the evaluations folder and moved it back to the deliveries. Also changed the the link for the correct grant

**Comment by keeganquigley** [comment]
Thanks @paulogustavopeixoto much appreciated. Moving on, I have the following comments:

- The article link gives a 404 error. Can you fix this?
- Getting an error when running `sqd build` about not having Axios:
```rust
src/Astar/mappings/utils/common.ts:2:19 - error TS2307: Cannot find module 'axios' or its corresponding type declarations.

2 import axios from 'axios';
```
I'm able to fix it by running `npm install axios --save`. I think you just need to declare the dependency in `package.json`

- Lastly, there are some errors in the unit tests currently. Can you take a look at my [evaluation](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/evaluations/liisa-portfolio-tracker_2_keeganquigley.md) and provide fixes for these? Thanks!

**Comment by keeganquigley** [comment]
Additionally, the React script and tests for the frontend can't be found even after the first steps are completed:

```ts
sh: 1: react-scripts: not found
ubuntu@ip-172-31-27-207:~/PolkadotNFTPortfolioTracker/frontend$ nvm use node
Now using node v22.2.0 (npm v10.7.0)
ubuntu@ip-172-31-27-207:~/PolkadotNFTPortfolioTracker/frontend$ npm start

> liisa-dashboard@0.1.0 start
> react-scripts start

sh: 1: react-scripts: not found
ubuntu@ip-172-31-27-207:~/PolkadotNFTPortfolioTracker/frontend$ npm test

> liisa-dashboard@0.1.0 test
> jest --coverage

sh: 1: jest: not found
```

**Comment by paulogustavopeixoto** [comment]
hey @keeganquigley . Let me have a look at those and I will get back to you

**Comment by paulogustavopeixoto** [comment]
Hello @keeganquigley , I made some adjustments to the readme file and instructions on how to test the frontend. Can you test it and let me know on your side if it is working? Its working fine on my side.

File: deliveries/LiisaPortfolioTracker_2.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+| 0c.    | Testing Guide             |  [README](https://github.com/LiisaNFT/PolkadotNFTPortfolioTracker/blob/19a8574c491af2a682226618a204a30dbd1f5497/README.md#getting-started) | ... |
+| 0d.    | Docker |  [Dockerfile](https://github.com/LiisaNFT/PolkadotNFTPortfolioTracker/blob/30be68d4894d9f09262296e2986f18eb11a6f2c4/multichain-indexer/Dockerfile)| ...|
+| 0e.    | Article |  (https://medium.com/@liisanft/introducing-liisas-multi-chain-portfolio-tracker-on-polkadot-5aeaaa334d88)| ...|
+| 1a. | Bubble.io wallet plugins |  (https://bubble.io/contributor/1556362159556x895666664689713800)| ...|
+| 1b. | Wallet address transmission |  [app](https://github.com/LiisaNFT/PolkadotNFTPortfolioTracker/blob/main/frontend/src/App.js)| ...|
+| 2a. | Dashboard Structure Implementation |  [src](https://github.com/LiisaNFT/PolkadotNFTPortfolioTracker/tree/main/frontend/src)| ...|
+| 2b. | Dynamic Dashboard Values |  |[functions](https://github.com/LiisaNFT/PolkadotNFTPortfolioTracker/tree/main/multichain-indexer/src/functions)| ...|
+
+**Additional Information**
+> We pretend to maintain the code and keep doing improvements on it even after the conclusion of this grant.
```
Comment (keeganquigley) [inline deliveries/LiisaPortfolioTracker_2.md:26]: I'm assuming rather than "pretend", you meant "plan" to maintain the code?

**Comment by paulogustavopeixoto** [comment]
hey @keeganquigley . I am currently having a look at this and will get back to you asap.

**Comment by paulogustavopeixoto** [comment]
hey @keeganquigley I have updated the delivery file and altered the text on it as per your suggestion. In regards to the error found on the testing part of the code I fixed and committed the changes to the repository.

**Comment by keeganquigley** [comment]
Thanks for the changes @paulogustavopeixoto much appreciated. Everything is working now and I'm happy to pass the milestone. Here is my [final evaluation](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/evaluations/liisa-portfolio-tracker_2_keeganquigley.md). Congrats on completion of the grant!

**Pull request merged**